src/rgw/driver/rados/rgw_sal_rados.cc

```diff
@@ -3482,6 +3477,101 @@ int RadosLuaManager::list_packages(const DoutPrefixProvider *dpp, optional_yield
   return 0;
 }
 
+int RadosLuaManager::watch_reload(const DoutPrefixProvider *dpp, uint64_t* watch_handle, librados::WatchCtx2* watcher)
```

Comment:
could not make that a sal API since the callbacks are expected to belong to a class deriving from `WatchCtx2` which is RASDOS specific